viosock/sys/viosock.inx

```diff
@@ -26,7 +26,7 @@ DriverPackageDisplayName = %VirtioSocket.DeviceDesc%
 PnpLockdown = 1
 
 [DestinationDirs]
-DefaultDestDir = 12
+DefaultDestDir = 13
 VirtioSocket_Lib_CopyFiles = 11
```

Comment:
According to https://learn.microsoft.com/en-us/windows-hardware/drivers/develop/run-from-driver-store#other-files, you should also move VirtioSocket_Lib_CopyFiles to 13. Please check if is this possible or not. This does not work for RNG providers.

Reply:
I will look into it, thanks. Yes, I know about RNG.

Reply:
@MartinDrab Please look into changes in https://github.com/virtio-win/kvm-guest-drivers-windows/pull/1075
HLK for 2022 and attestation signing fails with DIRID13 and without TargetOS decoration